src/main/java/Entities/Playlist.java

```diff
@@ -37,7 +43,19 @@ public boolean returnPrivacy(){
         return this.isPublic;
     }
 
-    public void addSong(Song newSong){
-        this.songList.add(newSong);
+    public ArrayList<User> showCollaborators() {
+        return this.collaborators;
+    }
+
+    public LinkedList<Song> showSongList(){
```

Comment:
Minor comment but is this a getter? if so getCollaborators() might be more consistent w other people's stuff

Reply:
Ill change it to getCollaborators() now then, it is a getter.